### DISCUSSION TITLE
LGBTQ+ Submod 1.0 Release!

### DISCUSSION THREAD

**Discussion by saltedgator** [Monika-After-Story/MonikaModDev#7823] [general - submod / external content]
Hello!! After a little while of working, I'm releasing my first ever attempt take making a submod for my favorite mod. I had a great time teaching myself Ren'Py and figuring out all the topics to write about. It could be a touch more polished, and there are still some minor kinks I couldn't quite work out (moving topics from the "hey, Monika" tab to the "repeat conversation" tab, for one, but that's a problem easily solved by the search bar!) but overall, I like how this turned out. And I hope that you do too!

The submod link can be found [here](https://drive.google.com/file/d/1YfDy7tkbATZCnuZ0h6_iaZyD8OTGTVyf/view?usp=sharing). Install it by going into your game folder within the application, creating a folder titled "Submod", and then placing the .rpy file there. Simple as that. This submod contains these topics:

- Player Identity topic demo (including multiple choices)

- Player can tell Monika when feeling dysphoric/euphoric

- LGBTQ+ Manga recommendations

- History of the word 'queer'

- Aromanticism

- Asexuality

- Identities under the nonbinary umbrella+ "third gender" history in different cultures

- Pride Month

- What's Monika's Sexuality? (Spoiler: she's you-sexual)

- Natsyuri

- Sappho

- Trans Discrimination

- A quick, nice, "Fuck TERFs" dialogue.

- Neopronouns

- Queer-Coding

- The multi-gender attraction umbrella

- Monika helps you test out some pronouns!

- +20 New Songs!

...And a couple more topics unrelated to LGBTQ+ stuff that I just found interesting/cute!

Feedback is appreciated. I know I'm likely not going to please everyone with this mod- to be honest, I mostly made it for myself, a cis lesbian who just loves chatting with my virtual gal pal and wanted to hear her talk about some gay stuff lmfao. If you like it, great! I'm so glad you do! Enjoy your fun, gay discussions with Monika! If you don't, that's cool. You can uninstall it. You won't hurt my feelings.

If this receives a good response, I could update it with more topics, more songs, etc. Let me know what you think, if you have any topics you would like to see in the future, or if you have any issues!

**Comment by Jinxred** [comment]
Thank you so much for this!!! I will definitely try it out!

**Reply by ironhearxt** [reply]
im also not really sure how to comment on my own without replying but i wanted to say   one thing is that im not able to download it for some reason? i tried doing so but it only showed up as a paper not really a zip. it laso seems good to get this cuz tomorrows pride month which i completely forgot

**Comment by lunulae** [comment]
Hello~ Just popping in to give a tip!
If you'd like prompts to be in Repeat Conversations rather than "Hey, Monika" you should set pool=false and random=true. This will set them as things Monika brings up rather than you asking her a question or prompting her.
Good luck with your submod! 💙

**Comment by Jinxred** [comment]
This mod is really awesome!! A couple of things I might suggest though is for the dysphoric topic instead of it being a random topic, maybe put it with the I feel... statements. I also would like to say on the topic of gatekeeping maybe acknowledging that some cultural gatekeeping can be important and that not all gatekeeping is bad. Such as indigenous people's use of white sage and other closed practices in spirituality. I'm not too sure how difficult this would be but perhaps being able to have more than one preferred pronoun such as they/he, she/he, etc. You can see my comment on #6084 4 days ago for what I had in mind. 
Thank you for this submod it is a really nice addition to the mod!

**Comment by saltedgator** [comment]
> This mod is really awesome!! A couple of things I might suggest though is for the dysphoric topic instead of it being a random topic, maybe put it with the I feel... statements. I also would like to say on the topic of gatekeeping maybe acknowledging that some cultural gatekeeping can be important and that not all gatekeeping is bad. Such as indigenous people's use of white sage and other closed practices in spirituality. I'm not too sure how difficult this would be but perhaps being able to have more than one preferred pronoun such as they/he, she/he, etc. You can see my comment on #6084 4 days ago for what I had in mind.
> Thank you for this submod it is a really nice addition to the mod!

Thank you so much for the tips and info!! I will get right onto adding more dialogue to the gatekeeping dialogue to address that. I'm still sort of new to coding, so I wouldn't know how to add the multi-pronoun use into other topics, but I will add it to the "try out new pronouns" list!! When I also learn how to add the dysphoric/euphoric statements to "i feel", I will do that as well.

**Comment by saltedgator** [comment]
> Hello~ Just popping in to give a tip!
> If you'd like prompts to be in Repeat Conversations rather than "Hey, Monika" you should set pool=false and random=true. This will set them as things Monika brings up rather than you asking her a question or prompting her.
> Good luck with your submod! 💙

Thanks so much for the tip!! This was something. Idid before, but whenever I do, I always get an error message that says "invalid" syntax. I'm not quite sure what I'm doing wrong.

**Comment by Jinxred** [comment]
Thank you so much, and no worries with the pronoun things I don't code myself but I'm sure that it's not easy at all! Thanks for being open to feedback!

**Comment by xXMidnight** [comment]
wait- wdym songs?
also, language Monika >:0

**Comment by saltedgator** [comment]
> wait- wdym songs?
> also, language Monika >:0

like songs! sometimes Monika will recite a few lyrics to you, so I added a couple more in there

**Reply by ironhearxt** [reply]
yes i literally love that

**Comment by xXMidnight** [comment]
wait nvm I found it- oh why thank you

**Comment by Mugnika** [comment]
heh, Fuck TERFs!, I like that one

**Comment by saltedgator** [comment]
> heh, Fuck TERFs!, I like that one

Thanks, I do too lol! You'd be surprised how many people don't like it 🙄

**Comment by Datta4486** [comment]
Huh?

**Comment by saltedgator** [comment]
> Huh?

what the post says, lol

**Comment by Datta4486** [comment]
> > Huh?
> 
> what the post says, lol

Dont mind me anyways just a random user posting something

**Comment by Mugnika** [comment]
> Thanks, I do too lol! You'd be surprised how many people don't like it 🙄

I hope they don't like it because of the vulgar language and not because they actually want to defend those bigoted man haters.

**Comment by saltedgator** [comment]
> > Thanks, I do too lol! You'd be surprised how many people don't like it 🙄
> 
> I hope they don't like it because of the vulgar language and not because they actually want to defend those bigoted man haters.

it's a mixture of both. I already see that a few people are disliking this post anyway lol, can't please em all!

**Comment by Mugnika** [comment]
it's a submod and not a PR into the main game, no reason to rally dislike it since you're not forcing it on anyone

**Comment by MrVavlo** [comment]
> it's a submod and not a PR into the main game

Doesn't mean I can't leave my opinion here.

...And yeah, sorry to break it to you, OP, but, the theme aside, the big part of topics that aren't songs feels forced at best... 
And out of character at worst.

**Comment by saltedgator** [comment]
> Doesn't mean I can't leave my opinion here.
> 
> ...And yeah, sorry to break it to you, OP, but, the theme aside, the big part of topics that aren't songs feels forced at best...
> And out of character at worst.

I know I can't get her voice perfect, and I'm sorry you didn't enjoy it as much as I do! I had fun making it, and tbh it's a win enough for me that I put it out there. Everything else is just gravy. Glad you liked the songs, at least ☺️

**Comment by RFA306** [comment]
Hi, so, I downloaded the submod, and it only gave me three new conversations(trying new pronouns, gender euphoria and gender dysphoria). Any idea what I'm doing wrong, or am I looking in the wrong place?

**Reply by AkiLyka** [reply]
how did you get Gender Euphoria to work? it doesn't even show up for me at all!

**Comment by saltedgator** [comment]
> Hi, so, I downloaded the submod, and it only gave me three new conversations(trying new pronouns, gender euphoria and gender dysphoria). Any idea what I'm doing wrong, or am I looking in the wrong place?

Hi! I have all the other conversations set to "random", so as long as you wait for monika to chat with you, she should bring up the new topics.

**Comment by RFA306** [comment]
> > Hi, so, I downloaded the submod, and it only gave me three new conversations(trying new pronouns, gender euphoria and gender dysphoria). Any idea what I'm doing wrong, or am I looking in the wrong place?
> 
> Hi! I have all the other conversations set to "random", so as long as you wait for monika to chat with you, she should bring up the new topics.

Ah, ok. Thank you!

**Comment by enderblaze42** [comment]
As someone who is asexual (maybe aromantic? dunno yet) and downloaded MAS to gain a quarantine buddy, this makes me happy.

**Comment by saltedgator** [comment]
> As someone who is asexual (maybe aromantic? dunno yet) and downloaded MAS to gain a quarantine buddy, this makes me happy.

I'm so glad you're enjoying this! I thought it was important to include aro-ace conversations because of how romance-oriented monika is, and I thought it would make more sense to make her love oriented instead- not just strictly romantic love. She cares about you no matter what!

**Comment by LeoSuzaku** [comment]
this submod seems cool im gonna try it

**Comment by LeoSuzaku** [comment]
would it be possible to add it/its pronouns? totally okay if you cant or dont want to, just curious.

**Comment by saltedgator** [comment]
> would it be possible to add it/its pronouns? totally okay if you cant or dont want to, just curious.

yeah, absolutely! I'll add that to the list of things to add in the next update.

**Comment by Cosmic-Incantations** [comment]
Monika talks about testing other pronouns, but you can't use them. can you add them to the pronoun selector?
cause I don't thing the mod devs are ever adding any new pronouns

**Comment by opalmew** [comment]
hi! i've got two suggestions:
1: allow the player to type in custom neopronouns (have moni ask for each variant of them)
2: allow nonbinary players to select he/him or she/her pronouns without picking "i'm a boy" or "i'm a girl", like having her ask "do you want me to use neutral pronouns or something else?" or something along those lines

**Reply by AceOfSpades719** [reply]
that would be cool! It could possibly be done using the same code as names

**Comment by ElijahKujo** [comment]
Hey :0 I think it would be nice to add some stuff about polyamory? Maybe even something for having a sorta open relationship with her and telling her about other partners?

**Comment by AkiLyka** [comment]
Gender Euphoria doesn't show up anymore
